html-lists/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <h2>Ordered List Example</h2>
+    <ol>
+      <li>Learn to code.</li>
+      <li>Build projects.</li>
+      <li>Get a job.</li>
+    </ol>
+    <h3>Unordered List Example</h3>
```

Comment:
Just a side note, this is probably better as a `<h2>` tag instead of `<h3>` since it is on the same level as the other `<h2>` for Ordered List Examples.